cmake/modules/FindOracle.cmake

```diff
@@ -42,9 +42,19 @@ find_path(ORACLE_INCLUDE_DIR
   # instant client from rpm
   /usr/include/oracle/*/client${LIB_SUFFIX})
 
+set(ORACLE_OCI_MIN_VERSION 10)
+set(ORACLE_OCI_MAX_VERSION 19)
+message(STATUS "ORACLE_OCI_MIN_VERSION ${ORACLE_OCI_MIN_VERSION} , ORACLE_OCI_MAX_VERSION ${ORACLE_OCI_MAX_VERSION}")
+message(STATUS "If your oracle client version is not in diapason between ${ORACLE_OCI_MIN_VERSION} and ${ORACLE_OCI_MAX_VERSION} probably
```

Comment:
I'd say "range" instead of "diapason", but mostly I wonder if we really want to always give this message. Maybe do it only if Oracle libraries are not found?

Reply:
I will try to do as you suggested in the next two days